src/openloco/windows/vehicle_list.cpp

```diff
@@ -0,0 +1,62 @@
+#include "../interop/interop.hpp"
+#include "../openloco.h"
+#include "../ui/WindowManager.h"
+#include <stdexcept>
+
+using namespace openloco::interop;
+
+namespace openloco::ui::windows::vehicle_list
+{
+    enum widx
+    {
+        frame = 0,
+        caption = 1,
+        close_button = 2,
+        panel = 3,
+        tab_trains,
+        tab_buses,
+        tab_trucks,
+        tab_trams,
+        tab_aircraft,
+        tab_ships,
+        company_select,
+        sort_name,
+        sort_profit,
+        sort_age,
+        sort_reliability,
+        scrollview,
+    };
+
+    // 0x004C19DC
+    window* open(uint16_t playerId, uint8_t type)
```

Comment:
There's a shortcut for this one as well (`0x004BF2D1`)

Reply:
Bit outside the scope of where I want to take this PR. Seems like something for #271 to tackle.